src/runtime/store/index.ts

```diff
@@ -207,6 +207,17 @@ export function derived<T>(stores: Stores, fn: Function, initial_value?: T): Rea
 	});
 }
 
+/**
+ * Takes a store and returns a new one derived from the old one that is readable.
+ *
+ * @param store - store to make readonly
+ */
+export function readonly<T>(store: Readable<T>): Readable<T> {
+	return {
+		subscribe: store.subscribe
```

Comment:
We should play it safe and bind the function in case someone passes in a custom store where subscribe has a this context

```suggestion
		subscribe: store.subscribe.bind(store)
```